test/ogc-api-coverages/get-coverage-rangeset-shapefile.ts

```diff
@@ -277,7 +278,7 @@ describe('OGC API Coverages - getCoverageRangeset with shapefile', function () {
     describe('when the CMR returns a 4xx', function () {
       const cmrErrorMessage = 'Corrupt GeoJSON';
       const cmrStatus = 400;
-      hookCmr('cmrPostSearchBase', { status: cmrStatus, data: { errors: [cmrErrorMessage] } });
+      hookCmr('queryGranulesForCollection', () => { throw new CmrError(cmrStatus, cmrErrorMessage); });
```

Comment:
The only reason I made this change is because queryVariables also uses POST, we need queryVariables to be able to call the non-stubbed version of cmrPostSearchBase().